### PR TITLE
Events update 160951919

### PR DESCRIPTION
**Pull request by chibujax**
#### What Does This PR Do?
- Updates Events

#### Description Of Task To Be Completed
- Add Creator to events pay load
- Populates Events form
- Updates event
- Update UploadFile and CustomDropDown components
#### How can this be manually tested?
- Click on any event that you are the creator then click on the edit button
#### What are the relevant pivotal tracker stories?
- #160951919
#### Screenshot(s)
screen shot 2018-10-05 at 1 40 30 pm
screen shot 2018-10-05 at 1 40 15 pm

### CODE REVIEW

**Comment by andela-sjames** [comment]
What's happening here... Not ready? @chibujax

**Comment by chibujax** [comment]
> What's happening here... Not ready? @chibujax

Ready but I want to fix some of the fixable code climate issues

File: client/pages/Event/EventDetailsPage.jsx [hunk 1]
```diff
@@ -103,7 +144,48 @@ class EventDetailsPage extends React.Component {
     );
   };
 
+  loadEvent() {
+    const { match: { params: { eventId } } } = this.props;
+    const { getEventAction } = this.props;
+    getEventAction(eventId);
```
Comment (kenneth-kip) [inline client/pages/Event/EventDetailsPage.jsx]: This is missing in prop validation.

File: client/pages/Event/EventDetailsPage.jsx [hunk 2]
```diff
@@ -10,40 +10,81 @@ import NotFound from '../../components/common/NotFound';
...
     return (
       <div className="event-details__top">
         <div className="event-details__section">
           <div className="event-details__event_title">{title}</div>
           <div className="event-details__social_event">{socialEvent.name}</div>
-          <button type="submit" className="event-details__rsvp_button">
-            {' '}
-            RSVP &#10004;
-          </button>
+          {creator ?
```
Comment (kenneth-kip) [inline client/pages/Event/EventDetailsPage.jsx]: Eslint is complaining here.

File: client/reducers/eventReducers.js [hunk 3]
```diff
@@ -35,19 +35,16 @@ export const events = (state = initialState.events, action) => {
...
+      const stateFormat = state.events || state;
+      const updated = stateFormat.map((item) => {
+        let newItem = {};
+        if (item.node.id === action.payload.updateEvent.updatedEvent.id) {
+          newItem = { node: action.payload.updateEvent.updatedEvent };
+          return newItem;
+        }
+        return item;
+      });
+      return { events: updated, status: 'updated' };
```
Comment (kenneth-kip) [inline client/reducers/eventReducers.js]: Line break after closing and opening braces.

File: client/pages/Event/EventDetailsPage.jsx [hunk 4]
```diff
@@ -10,40 +10,81 @@ import NotFound from '../../components/common/NotFound';
...
 /**
  * @description Currently contains an event details page layout
  *
  * @class EventDetailsPage
  * @extends {React.Component}
  */
 class EventDetailsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    const { events } = this.props;
```
Comment (kenneth-kip) [inline client/pages/Event/EventDetailsPage.jsx]: Events missing in prop validation

**Pull request merged**